src/HW_models/NRF_DPPI.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2020 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef _NRF_HW_MODEL_DPPI_H
+#define _NRF_HW_MODEL_DPPI_H
+
+#include "nrfx.h"
+
+#ifdef __cplusplus
+extern "C"{
+#endif
+
+extern NRF_DPPIC_Type NRF_DPPI_regs;
```

Comment:
The declaration `extern NRF_DPPIC_Type NRF_DPPI_regs;` can be moved to the c file

Reply:
It is placed in the h file for almost all other peripherals, so I placed it there for consistency